src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -59,12 +59,29 @@ public void Register(Assembly assembly)
             EnvironmentSettings.SettingsLoader.Components.RegisterMany(assembly.GetTypes());
         }
 
+        [Obsolete("Use GetTemplates instead")]
```

Comment:
Can we use `nameof(GetTemplates)` here to allow auto-refactoring later?

Reply:
interpolated strings are not supported in this context